### PR TITLE
Added Time Delays from OM10 into sprinkled objects

### PR DESCRIPTION
**Pull request by jbkalmbach**
By adjusting the variability parameters string from catsim database we can now add in time delays to the variability of our lensed AGN. This takes care of #47

### CODE REVIEW

**Comment by drphilmarshall** [comment]
Oh yeah! Nice work Bryce :-) Have you got a validation plot that shows the delay between two quasar image sources in a nightly sampled reference light curve? Post it here so we can admire it! :-)

File: twinkles/twinklesCatalogDefs.py [hunk 1]
```diff
@@ -48,3 +48,37 @@ def get_phoSimMagNorm(self):
                 magNorm_out = magNorm + self.column_by_name(varName)
 
             return magNorm_out
+
+    def _query_and_write(self, filename, chunk_size=None, write_header=True,
```
Comment (danielsf) [inline twinkles/twinklesCatalogDefs.py]: This does not look like it is any different from the ```_query_and_write()``` method defined in ```InstanceCatalog```.  Why did you reimplement it?

**Comment by danielsf** [comment]
@jbkalmbach Obviously, I would write a unit test to make sure that  everything comes out the way you expect, but I agree with you that this is how the variability method registry is supposed to work.

File: twinkles/twinklesCatalogDefs.py (hunk 1 above)
Comment (jbkalmbach) [inline twinkles/twinklesCatalogDefs.py]: Oh, I will remove that. That was from an earlier attempt to get this to work. Thanks.

**Comment by jbkalmbach** [comment]
So, I made a toy om10 catalog and then ran a set of daily observations. This gave me a set of phosim catalogs with the objects and I just plotted them in an[ ipython notebook](http://nbviewer.jupyter.org/gist/jbkalmbach/f75eefddb39cb4906bac). I'm not sure if we're going to add an example or test directory then I could add them there, but I've also attached the files to recreate the notebook here if anybody wants to try it.

[testGeneratePhosimInput.py](https://files.zenhub.io/56b2851b7781466354264573)
Uploaded using [**ZenHub.io**](http://www.zenhub.io)
[testOM10.fits](https://files.zenhub.io/56b2851c7781466354264574)
Uploaded using [**ZenHub.io**](http://www.zenhub.io)

**Comment by drphilmarshall** [comment]
Great! I see delays alright :-) We'll probably want to repeat validation
tests like this on the real OM10 systems (ie look at more plots), to
convince ourselves that the light curves are being realized correctly. Nice
light curves!

**Comment by cdfassnacht** [comment]
Hi Bryce,

This looks really nice, and it's fun to be able to see the delays by eye.
Thanks for sending it around.

   Chris

On Wed, Feb 3, 2016 at 3:53 PM, Phil Marshall <notifications@github.com>
wrote:

> Great! I see delays alright :-) We'll probably want to repeat validation
> tests like this on the real OM10 systems (ie look at more plots), to
> convince ourselves that the light curves are being realized correctly. Nice
> light curves!
>
> On Wed, Feb 3, 2016 at 2:55 PM, Bryce Kalmbach <notifications@github.com>
> wrote:
>
> > So, I made a toy om10 catalog and then ran a set of daily observations.
> > This gave me a set of phosim catalogs with the objects and I just plotted
> > them in an ipython notebook
> > <http://nbviewer.jupyter.org/gist/jbkalmbach/f75eefddb39cb4906bac>. I'm
> > not sure if we're going to add an example or test directory then I could
> > add them there, but I've also attached the files to recreate the notebook
> > here if anybody wants to try it.
> >
> > testGeneratePhosimInput.py
> > <https://files.zenhub.io/56b2851b7781466354264573>
> > Uploaded using *ZenHub.io* <http://www.zenhub.io>
> > testOM10.fits <https://files.zenhub.io/56b2851c7781466354264574>
> > Uploaded using *ZenHub.io* <http://www.zenhub.io>
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/DarkEnergyScienceCollaboration/Twinkles/pull/107#issuecomment-179516894
> >
> > .
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DarkEnergyScienceCollaboration/Twinkles/pull/107#issuecomment-179537885>
> .
>



-- 
Chris Fassnacht
Professor, Physics Dept.
UC Davis
1 Shields Ave.
Davis, CA 95616
+1-530-554-2600

**Comment by jbkalmbach** [comment]
I'm going to go ahead and merge this and make a new issue for validation.

**Comment by drphilmarshall** [comment]
Woot! :-)

**Pull request merged**